packages/leancode_lint/test/lints_test_app/lib/use_dedicated_media_query_methods_test.dart

```diff
@@ -0,0 +1,128 @@
+// Ignore for test purposes
+// ignore_for_file: unnecessary_statements, deprecated_member_use
+
+import 'package:flutter/material.dart';
+
+class FooWidget extends StatelessWidget {
+  const FooWidget({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    // dart format off
+    // expect_lint: use_dedicated_media_query_methods
+    final width = MediaQuery.of(context).size
+    .width;
+    // dart format on
+
+    // expect_lint: use_dedicated_media_query_methods
+    final padding = MediaQuery.of(context).padding;
```

Comment:
Maybe this lint should be reported only on the `of` and `maybeOf`, instead of the entire expression?

Reply:
I don't think the range of the affected code is of concern 🤔 Usually, you report the lint on a single element that's in question